From a8ad18614432dfcabb830f78ff2ca41297fdc33b Mon Sep 17 00:00:00 2001
From: Steve Sakoman <steve@sakoman.com>
Date: Fri, 9 Jul 2010 14:51:13 -0700
Subject: [PATCH 31/31] Revert "omap2_mcspi: Flush posted writes"

This reverts commit a330ce2001b290c59fe98c37e981683ef0a75fdf.

The reverted commit breaks the ADS7846 driver
---
 drivers/spi/omap2_mcspi.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/spi/omap2_mcspi.c b/drivers/spi/omap2_mcspi.c
index b3a94ca..83b5592 100644
--- a/drivers/spi/omap2_mcspi.c
+++ b/drivers/spi/omap2_mcspi.c
@@ -204,7 +204,6 @@ static inline void mcspi_write_chconf0(const struct spi_device *spi, u32 val)
 
 	cs->chconf0 = val;
 	mcspi_write_cs_reg(spi, OMAP2_MCSPI_CHCONF0, val);
-	mcspi_read_cs_reg(spi, OMAP2_MCSPI_CHCONF0);
 }
 
 static void omap2_mcspi_set_dma_req(const struct spi_device *spi,
-- 
1.7.1

